bedrock/mozorg/templates/mozorg/home/home-b.html

```diff
@@ -0,0 +1,274 @@
+      <div class="slider-cover">
+        <div class="slider-content">
+          <p class="slider-intro">
+            {{ _('We are a global network of educators, builders, experts and advocates.') }}
+          </p>
+
+          <div class="toggle-content">
+            <div class="toggle-buttons">
+              <button class="toggle-open toggle-who">{{ _('Our impact') }}</button>
+              <button class="toggle-close toggle-who"><span>{{ _('Close') }}</span></button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="slider-detail">
+        <div class="slider-content">
+          <h2 class="slider-detail-headline">{{ _('Our impact') }}</h2>
+          <p>
+          {% trans %}
+            Working at the grassroots and policy levels, we teach web literacy,
+            provide tools and advocate on behalf of every individual who values
+            an internet built on fairness, inclusion and respect.
+          {% endtrans %}
+          </p>
+
+          <ul class="icon-links">
+            <li id="encryption"><a href="https://advocacy.mozilla.org/encrypt/">{{ _('Stand Up For Encryption') }}</a></li>
+            <li id="teachtheweb"><a href="https://learning.mozilla.org/">{{ _('Teach The Web') }}</a></li>
+            <li id="leandata"><a href="{{ url('mozorg.about.policy.lean-data') }}">{{ _('Try the Lean Data Toolkit') }}</a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
+
+    <section class="slider-wrapper" id="innovate">
+      <div class="slider-cover">
+        <div class="slider-content">
+          <p class="slider-intro">
+            {{ _('We build web technologies designed to excite, inspire and promote openness.') }}
+          </p>
+
+          <div class="toggle-content">
+            <div class="toggle-buttons">
+              <button class="toggle-open toggle-innovate">{{ _('Our innovations') }}</button>
+              <button class="toggle-close toggle-innovate"><span>{{ _('Close') }}</span></button>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="slider-detail">
+        <div class="slider-content">
+          <h2 class="slider-detail-headline">{{ _('Our innovations') }}</h2>
+          <p>
+          {% trans %}
+            Using the web as the platform, we build open, innovative
+            technologies that allow developers to work free of closed, corporate
+            ecosystems and create faster, safer web experiences for us all.
+          {% endtrans %}
+          </p>
+
+          <ul class="icon-links">
+            <li id="gaming"><a href="https://games.mozilla.org/">{{ _('Gaming on the Web') }}</a></li>
+            <li id="vr"><a href="https://aframe.io/">{{ _('Virtual Reality Platform') }}</a></li>
+            <li id="servo"><a href="https://servo.org/">{{ _('Servo') }}</a></li>
+            <li id="rust"><a href="https://www.rust-lang.org/">{{ _('Rust') }}</a></li>
```

Comment:
Rust... On the home page!

![rust](https://cloud.githubusercontent.com/assets/400117/16653703/10cd548e-444a-11e6-8e58-1248db19c330.gif)